src/widgets/scrollbar.rs

```diff
@@ -29,6 +29,11 @@ use crate::{prelude::*, symbols::scrollbar::*};
 /// └─────────── begin
 /// ```
 ///
+/// ## Important
+/// When rendering `Scrollbar`, you must specify the `content_length`
+/// field in the [`ScrollbarState`], or else the `Scrollbar` will not render. See
+/// the documentation for [`ScrollbarState`] for more information.
+///
```

Comment:
Small formatting issue: Can you please rewrap this to 100 chars (I use https://marketplace.visualstudio.com/items?itemName=stkb.rewrap, but there are similar tools for most editors / IDEs), change the `##` to `#` and put a blank line after the heading?

Reply:
I handled this and simplified the wording a bit.